### PR TITLE
[4000] - Empty Cart on Reorder

### PR DESCRIPTION
**Pull request by leonazava**
**Related issue(s):**
* Fixes scandipwa/scandipwa/issues/4000

**Problem:**
* The cart wasn't displaying loader when a user clicked Reorder from their profile

**In this PR:**
* Updated CartPage.component.js by adding an isLoading state from store and extracting the loader rendering logic into a new renderInitialPlaceholder method

### CODE REVIEW

**Comment by aleksandrakorolova** [comment]
Issues from #4504:
2. Fixed
5. Fixed
6. Fixed
7. A lot of orders are displayed when changing quantity of product in the cart
https://drive.google.com/file/d/10gIMDxljLJpURCHtu2fkiKSzJjNUKJho/view?usp=sharing
8. Mobile: Loader is displayed not in the center of page
Screenshot 2022-03-28 at 15 20 21

File: packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss [hunk 1]
```diff
@@ -273,6 +273,10 @@
     }
 
     .Loader {
+        @include mobile {
+            visibility: hidden;
```
Comment (zans-laksa) [inline packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss:277]: This makes an element invisible, but it still takes place in the layout.
Is this intended?

**Review by zans-laksa — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss (hunk 1 above)
Comment (leonazava) [reply packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss]: That was a part of an earlier implementation. Tested it with display: none and that indeed looks like a better alternative.

**Review by zans-laksa — APPROVED** [review]

**Comment by aleksandrakorolova** [comment]
Please, remove loader from summary block in the cart because there is different behavior when changing quantity of product and deleting it

**Review by zans-laksa — APPROVED** [review]

**Pull request merged**